website/docs/patterns/multi-select-patterns/partials/specifications/specifications.md

```diff
@@ -0,0 +1,43 @@
+## Anatomy
```

Comment:
I think it still makes sense to treat this as a table because it's a nice way to present the information. But I've removed the "required/optional" statements and made this a bit more general/broad.